proxy_map.go

```diff
@@ -950,7 +1052,14 @@ func (m *Map) addIndex(ctx context.Context, indexConfig types.IndexConfig) error
 	return err
 }
 
-func (m *Map) addEntryListener(ctx context.Context, flags int32, includeValue bool, key interface{}, predicate predicate.Predicate, handler EntryNotifiedHandler) (types.UUID, error) {
+func (m *Map) addEntryListener(
+	ctx context.Context,
+	flags int32,
+	includeValue bool,
+	key interface{},
+	predicate predicate.Predicate,
+	handler EntryNotifiedHandler,
+) (types.UUID, error) {
```

Comment:
Does the IDE format each parameter on its own line? If not, it's OK to have the method signature on a single line.

Reply:
I guess I did it for readability. Reverting.